index.html

```diff
@@ -5,22 +5,113 @@
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
+		<title>Homework Assignment 3</title>
 
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
-
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,800" rel="stylesheet">
 	</head>
 	<body>
-		<div id="container">
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+			<header class="clearfix">
+				<h1>Relaxr</h1>  
+				<nav class="alignright clearfix"> 
+					<ul class="clearfix"> 
+						<li>	
+							<a href="#">About</a>
+						</li>
+						<li>
+							<a href="#">FAQ</a>
+						</li>
+						<li>
+							<a href="#">Team</a>
+						</li>
+						<li>
+							<a href="#">Contact Us</a>
+						</li>
+						<li>
+							<a href="#">Blog</a> 
+						</li>
+					</ul>	 
+				</nav>
 			</header>
+
+			<div class="container clearfix">
+				<div class="main alignleft">
+					<div class="article">
+						<h3>How I implemented Relaxr in 2 weeks and changed my life</h3>
+
+						<img src="images/blog_photo1.jpg" alt="Rowers">
+						<p>
+						<span class="drop alignleft">L</span>
+						orem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+						tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+						quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+						consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+						cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+						proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+						<div class="readmore">Read more &gt;</div>
```

Comment:
This should probably be an anchor tag `<a>` since "Read more" will likely visit another page. In this case, you'd also need to set it to `display: block;` so that it behaves similarly to the `div` w/ respect to `text-align: right;`